fission/src/systems/physics/PhysicsSystem.ts

```diff
@@ -719,6 +721,45 @@ class PhysicsSystem extends WorldSystem {
         JOLT.destroy(this._joltBodyInterface)
         JOLT.destroy(this._joltInterface)
     }
+
+    /**
+     * Creates a ghost object and a distance constraint that connects it to the given body
+     * The ghost body is part of the LAYER_GHOST which doesn't interact with any other layer
+     * The caller is responsible for cleaning up the ghost body and the constraint
+     *
+     * @param id The id of the body to be attatched to and moved
+     * @returns The ghost body and the constraint
+     */
+
+    public CreateGodModeBody(id: Jolt.BodyID): [Jolt.Body, Jolt.Constraint] {
+        const body = this.GetBody(id)
+        const bodyPosition = body.GetPosition()
+        const ghostPosition = new THREE.Vector3(bodyPosition.GetX(), bodyPosition.GetY(), bodyPosition.GetZ())
+        const ghostBody = this.CreateBox(new THREE.Vector3(0.05, 0.05, 0.05), undefined, ghostPosition, undefined)
```

Comment:
```suggestion
        const ghostBody = this.CreateBox(new THREE.Vector3(0.05, 0.05, 0.05), undefined, JoltVec3_ThreeVector3(anchor), undefined)
```
Or something like that